### PR TITLE
Traitor synth lockdown changes

### PR DESCRIPTION
**Pull request by EmperorJon**
Prevents traitor synthetics from being locked down on the robotics console.
Reports that they have been locked down on the UI to whoever attempted it.
Notifies the synth that an attempt was made.

### CODE REVIEW

**Comment by zaredman** [comment]
@EmperorJon -- Preventing lockdown how? Traitor = Unslaved?

**Comment by Spookerton** [comment]
There are two vars that govern how a synth is locked down - canmove and lockcharge. canmove is a mechanical var that actually disallows movement, and lockcharge is a mostly display var that governs how their lockdown state is reported to the console. At the moment, all borgs set the two at the same time. This splits traitor borgs specifically so that their lockcharge is set, but not their canmove, meaning they report that they are locked down but are not.

File: code/game/machinery/computer/robot.dm [hunk 1]
```diff
@@ -91,15 +91,23 @@
 		if(!target || !istype(target))
 			return
 
-		message_admins("<span class='notice'>[key_name_admin(usr)] [target.canmove ? "locked down" : "released"] [target.name]!</span>")
-		log_game("[key_name(usr)] [target.canmove ? "locked down" : "released"] [target.name]!")
-		target.canmove = !target.canmove
-		if (target.lockcharge)
+		var/istraitor = target.mind.special_role == "Traitor"
```
Comment (Zuhayr) [inline code/game/machinery/computer/robot.dm]: ````traitors.is_antagonist(target.mind)````

**Comment by Zuhayr** [comment]
:+1:

**Pull request merged**